PhpStorm Protocol (Win)/run_editor.js

```diff
@@ -22,6 +27,10 @@ if (match) {
 		file_system = new ActiveXObject('Scripting.FileSystemObject'),
 		file = decodeURIComponent(match[1]).replace(/\+/g, ' '),
 		search_path = file.replace(/\//g, '\\');
+		
+	if (settings.project_basepath != '' && settings.project_pathalias != '') {
+		file = file.replace(settings.project_basepath, settings.project_pathalias);

```

Comment:
Maybe we should use RegEx here to replace only once from beginning of the path to avoid incorrect replacements, when source `project_basepath` is `/` for example).